Matplotlib_common/latex_labels.py

```diff
@@ -1,10 +1,16 @@
 var_labels = {
+
+  "lwp" : 'LWP [kg m$^{-2}$]',
+  "rwp" : 'RWP [kg m$^{-2}$]',
+  "cwp" : 'CWP [kg m$^{-2}$]',
+  "lwm" : 'LWM [kg]',
+  "cwm" : 'CWM [kg]',
+  "rwm" : 'RWM [kg]',
   "sgs_tke" : 'SGS TKE (Smagorinsky) [m$^{2}$ s$^{-2}$]',
   "sgs_tke_sd" : 'SGS TKE (Super-droplets) [m$^{2}$ s$^{-2}$]',
-  "lwp" : 'LWP [g m$^{-2}$]',
-  "rwp" : 'RWP [g m$^{-2}$]',
   "surf_precip" : 'Surface precip. [mm/day]',
   "acc_precip" : 'Accumulated precip. [mm]',
+  "acc_vol_precip" : 'Accumulated volume precipitation [m^3]',
```

Comment:
There's no such thing as volume precipitation.
Change key from acc_vol_precip to acc_precip_vol and label "Accumulated volume precipitation [m^3]" to "'Accumulated precip. [m^3]".